client/helpers/addnewminutes.js

```diff
@@ -0,0 +1,35 @@
+import { FlowRouter } from 'meteor/ostrio:flow-router-extra';
+import { Minutes } from '/imports/minutes';
+import { MinutesFinder } from '/imports/services/minutesFinder';
+import { handleError } from '/client/helpers/handleError';
+export class addMinutes{
+    addMinutes(msid,msob) {
```

Comment:
Hi, the parameter msid is not needed. It is embedded inside msob!
Please rename msob to `meetingSeriesObject`
Remember: if you don't provide API comments, please use "self explanatory" interface variables.